main.go

```diff
@@ -2,41 +2,58 @@ package main
 
 import (
 	"database/sql"
+	"flag"
 	"fmt"
 	"net/http"
 	"os"
+	"strconv"
+	"strings"
 	"time"
 
 	_ "github.com/mattn/go-sqlite3"
 	"github.com/op/go-logging"
 )
 
+const (
+	portDefault = "8080"
+)
+
 var logger = logging.MustGetLogger("commento")
 var db *sql.DB
 
+// The run environment for the application. Either dev, staging, or prod
+var runEnv string
+
 func main() {
-	err := loadDatabase("sqlite3.db")
-	if err != nil {
+	if err := loadDatabase("sqlite3.db"); err != nil {
 		die(err)
 	}
 
-	fs := http.FileServer(http.Dir("assets"))
+	// Parse command line options
+	var port = flag.String("port", portDefault, "port for commento service")
+	flag.Parse()
 
-	http.Handle("/assets/", http.StripPrefix("/assets/", fs))
-	http.HandleFunc("/", indexHandler)
-	http.HandleFunc("/create", createCommentHandler)
-	http.HandleFunc("/get", getCommentsHandler)
-
-	var port string
+        // Check that we get a valid port value
+	if _, err := strconv.ParseInt(*port, 10, 32); err != nil {
```

Comment:
if the port flag value check requires an int, maybe simply requesting an int flag would suffice `flag.Int("port",8080,"..")` 

also, this check doesn't really work since it just checks if its can be converted to int32, ie "0", and any other int32 passes. this check is not really necessary since `ListenAndServe` will fail appropriately on an invalid port